### DISCUSSION TITLE
Auto payment forwarding to another BTC Address

### DISCUSSION THREAD

**Discussion by g33kme** [btcpayserver/btcpayserver#2424] [Plugins & integrations]
Would be great to have the possibility to setup an **auto payment forwarding service** for **hot wallets** to another btc address. Would be great to have a setup on the UI when clicking on "manage wallet"

Some config possibilites

- send instantly, every hour, every day ...
- full amount, specific amaoun
- receiving btc address
- set network fees manuall or with "confirm in the next 10min, 1h, 6h, 1day
- subtract fees from amount or not
- allow fee increase (RBF)

**Comment by ghost** [comment]
afaik, [BTC Transmuter](https://github.com/btcpayserver/btcTransmuter/) can do this already.
Though I think it will be depreciated with the Plugin system, so hopefully we'll have a plugin for such wallet functionality soon.

**Reply by Zaxounette** [reply]
AFAIK, I can confirm that the transmuter will, in time, be ported into a plugin.
No ETA.

**Reply by ghost** [reply]
> No ETA.

I, CEO of your BTCPay server, say that ETA is in #twoweeks!

**Comment by g33kme** [comment]
@MaxHillebrand Yes, I know about BTC Transmuter, but I think its a way to complicated. I think it would be better and more user friendly to have a simply option in the store hot wallet.

**Reply by Zaxounette** [reply]
As stated above, AFAIK, the transmuter will be ported into a plugin once the plugin feature is workable, refined and ready to use, enabling exactly what you're requesting for.
